_templates/layout.html

```diff
@@ -1,131 +1,162 @@
-  <meta name="twitter:image" content="{{ base_url }}/_static/{{ logo }}">
-  <meta name="twitter:site" content="@{{ twitter }}">
-  <meta name="twitter:title" content="{{ this_page_title }}">
-  <meta name="twitter:description" content="{{ description }}">
-  <meta property="og:title" content="{{ this_page_title }}">
-  <meta property="og:type" content="website">
-  <meta property="og:url" content="{{ base_url }}">
-  <meta property="og:site_name" content="{{ project }}">
-  <meta property="og:image" content="{{ base_url }}/_static/{{ logo }}">
-  <meta property="og:description" content="{{ description }}">
-
-  <!-- Plausible analytics for anonymous usage statistics -->
-  {{ plausible }}
-
-  {%- block extrahead %}{% endblock %}
-</head>
-
-
-<body>
-  {%- if alert is defined %}
-    <div class="alert-top alert alert-primary alert-dismissible fade show" role="alert">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    {%- if title == "Home" %} {% set this_page_title=project|striptags|e %} {%-
+    else %} {% set this_page_title=title|striptags|e + " | " +
+    project|striptags|e %} {%- endif %}
+    <title>{{ this_page_title }}</title>
+
+    <link rel="shortcut icon" href="{{ favicon_url|e }}" />
+
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="{{ pathto('_static/bootstrap-5.1/css/bootstrap.min.css', 1) }}"
+    />
+
+    <!-- FontAwsome icons -->
+    <link
+      rel="stylesheet"
+      href="{{ pathto('_static/css/fontawesome/css/all.css', 1) }}"
+    />
+
+    <!-- Pygments CSS for syntax highlighting -->
+    <!-- Accessible versions by Max Chadwick (@mpchadwick) -->
+    <link
+      rel="stylesheet"
+      href="{{ pathto('_static/css/pygments/native.css', 1) }}"
+    />
+
+    <!-- Custom CSS set in conf.py -->
+    <link rel="stylesheet" href="{{ pathto('_static/' + stylesheet, 1) }}" />
+
+    <!-- Social media cards -->
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:image" content="{{ base_url }}/_static/{{ logo }}" />
+    <meta name="twitter:site" content="@{{ twitter }}" />
+    <meta name="twitter:title" content="{{ this_page_title }}" />
+    <meta name="twitter:description" content="{{ description }}" />
+    <meta property="og:title" content="{{ this_page_title }}" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="{{ base_url }}" />
+    <meta property="og:site_name" content="{{ project }}" />
+    <meta property="og:image" content="{{ base_url }}/_static/{{ logo }}" />
+    <meta property="og:description" content="{{ description }}" />
+
+    <!-- Plausible analytics for anonymous usage statistics -->
+    {{ plausible }} {%- block extrahead %}{% endblock %}
+  </head>
+
+  <body>
+    {%- if alert is defined %} {%- for alert in alert %}
+    <div
+      class="alert-top alert alert-primary alert-dismissible fade show"
```

Comment:
It would be awesome if we could set the alert class from the `conf.py` to get different colours. Maybe if the `alert` variable was a list of tuples like `("alert-primary", "<i ....")`. 

Having the alert be a different colour when we change it helps make it pop when people visit the page.

Reply:
I could create and truple for the alert. The 0° element is the alert text and the 1° element change the color trough changing the style. The different color options are according to [bootstrap alerts](https://getbootstrap.com/docs/4.0/components/alerts/)

So let me know that you think about this

Reply:
Looks great!